set.mm

```diff
@@ -465654,6 +465751,432 @@ its graph has a given second element (that is, function value).
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+
+  Define a decimal expansion constructor.  The decimal expansions built with
+  this constructor are not meant to be used alone outside of this chapter.
+  Rather, they are meant to be used exclusively as part of a decimal number
+  with a decimal fraction, for example ` ( 3 . _ 1 _ 4 _ 1 _ 5 9 ) ` .
+
+  That decimal point operator is defined in the next section.
+  The bulk of these constructions have originally be proposed by David A.
+  Wheeler on 12-May-2015, and discussed with Mario Carneiro in this thread:
+  ~ https://groups.google.com/g/metamath/c/2AW7T3d2YiQ/m/iSN7g87t3ikJ .
+$)
+
+  $c _ $. $( Decimal fraction indicator for digits after decimal point $)
+
+  $( Constant used for decimal fraction constructor.  See ~ df-dp2 . $)
+  cdp2 $a class _ A B $.
+
+  $( Define the "decimal fraction constructor", which is used to build up
+     "decimal fractions" in base 10.  This is intentionally similar to
+     ~ df-dec .  (Contributed by David A. Wheeler, 15-May-2015.)  (Revised by
+     AV, 9-Sep-2021.) $)
+  df-dp2 $a |- _ A B = ( A + ( B / ; 1 0 ) ) $.
+
+  $( Obsolete version of ~ df-dp2 as of 9-Sep-2021.  (Contributed by David A.
+     Wheeler, 15-May-2015.)  (New usage is discouraged.)
+     (Proof modification is discouraged.) $)
+  dfdp2OLD $p |- _ A B = ( A + ( B / 10 ) ) $=
+    ( cdp2 c1 cc0 cdc cdiv co caddc c10 df-dp2 dec10OLD eqcomi oveq2i eqtri ) A
+    BCABDEFZGHZIHABJGHZIHABKQRAIPJBGJPLMNNO $.
+
+  $( Equality theorem for the decimal expansion constructor.  (Contributed by
+     David A. Wheeler, 15-May-2015.) $)
+  dp2eq1 $p |- ( A = B -> _ A C = _ B C ) $=
+    ( wceq c1 cc0 cdc cdiv co caddc cdp2 oveq1 df-dp2 3eqtr4g ) ABDACEFGHIZJIBO
+    JIACKBCKABOJLACMBCMN $.
+
+  $( Equality theorem for the decimal expansion constructor.  (Contributed by
+     David A. Wheeler, 15-May-2015.) $)
+  dp2eq2 $p |- ( A = B -> _ C A = _ C B ) $=
+    ( wceq c1 cc0 cdc cdiv co caddc cdp2 oveq1 oveq2d df-dp2 3eqtr4g ) ABDZCAEF
+    GZHIZJICBQHIZJICAKCBKPRSCJABQHLMCANCBNO $.
+
+  ${
+    dp2eq1i.1 $e |- A = B $.
+    $( Equality theorem for the decimal expansion constructor.  (Contributed by
+       David A. Wheeler, 15-May-2015.) $)
+    dp2eq1i $p |- _ A C = _ B C $=
+      ( wceq cdp2 dp2eq1 ax-mp ) ABEACFBCFEDABCGH $.
+
+    $( Equality theorem for the decimal expansion constructor.  (Contributed by
+       David A. Wheeler, 15-May-2015.) $)
+    dp2eq2i $p |- _ C A = _ C B $=
+      ( wceq cdp2 dp2eq2 ax-mp ) ABECAFCBFEDABCGH $.
+
+    dp2eq12i.2 $e |- C = D $.
+    $( Equality theorem for the decimal expansion constructor.  (Contributed by
+       David A. Wheeler, 15-May-2015.) $)
+    dp2eq12i $p |- _ A C = _ B D $=
+      ( cdp2 dp2eq1i dp2eq2i eqtri ) ACGBCGBDGABCEHCDBFIJ $.
+  $}
+
+  ${
+    dp20u.1 $e |- A e. NN0 $.
+    $( Add a zero in the tenths (lower) place.  (Contributed by Thierry Arnoux,
+       16-Dec-2021.) $)
```

Comment:
tenths ?
"Adding a zero as the least significant digit after a decimal separator does not change the value of a number." ?

Reply:
Yes, the tenths place! 
See for example [https://en.wikipedia.org/wiki/Numerical_digit#Computation_of_place_values](the Wikipedia for the place value). In french we might say "les unités, les dixièmes, les centièmes"